.env-oci-conformance

```
@@ -0,0 +1,14 @@
+export KEPPEL_API_PUBLIC_FQDN=localhost
+export KEPPEL_ISSUER_KEY=./privkey.pem
+export KEPPEL_DB_CONNECTION_OPTIONS=sslmode=disable
+export KEPPEL_DB_PASSWORD=mysecretpassword
+export KEPPEL_DB_PORT=54321
+export KEPPEL_USERNAME=keppel_e2e@default/master@ccadmin
+export KEPPEL_PASSWORD=37aded545238cd2414e25f5452322de85f7ca03863c4d09154
```

Since the `trivial` auth driver allows any values for this, we should use more dummy-esque values, for example:

```suggestion
export KEPPEL_USERNAME=johndoe
export KEPPEL_PASSWORD=5uper5ecret
```